docs/airdrop-eligibility-distribution.md

```diff
@@ -225,18 +223,16 @@ Scroll to the section that describes your scenario and follow the instructions.
 
 #### Scenario 3: You performed qualifying actions before and after Nitro was launched.
 
- 1. First, refer to the [qualifying actions section below](#qualifying-actions) to determine how many points you earned on Arbitrum One before Nitro launched. We'll refer to this as `OnePointsPre`.
- 2. Next, refer to the [point-to-token conversion table below](#point-to-token-conversion-table) and determine how many tokens you're eligible to claim based on the number of points you earned.
+ 1. First, refer to the [qualifying actions section](#qualifying-actions) to determine how many points you earned on Arbitrum One before Nitro launched. We'll refer to this as `OnePointsPre`.
+ 2. Next, refer to the [point-to-token conversion table](#point-to-token-conversion-table) and determine how many tokens you're eligible to claim based on the number of points you earned.
```

Comment:
suggested additional text:

2. Next, refer to the [point-to-token conversion table](#point-to-token-conversion-table) and determine how many tokens you're eligible to claim based on the number of points you earned. Convert those points into tokens.
3. Then, refer to the [qualifying actions section](#qualifying-actions) to determine how many points you earned on Arbitrum One **after** Nitro launched. We'll refer to this as `OnePointsPost`.
4. Next, refer to the [point-to-token conversion table](#point-to-token-conversion-table) and determine how many tokens you're eligible to claim based on the number of points you earned. Convert those points into tokens.
5. Now you should have token totals, based on points, calculated based on points scored before and after Nitro. Calculate the difference in points between the two totals, halve that, and add it to the token total calculated based on OnePointsPre. This is your final amount of eligible tokens.
6. In the case that your token total based on OnePointsPost was lower than the other total, you were awarded tokens solely based on OnePointsPre, as it was the higher amount.

Reply:
Silly q -> would you mind clarifying how, exactly, Nova considerations should be blended into this logic? It feels like the logic from Scenario 2 -> Step 2 above belongs somewhere in here, but I'm not confident about the calculus.

Providing a few examples of this path being executed, displayed in tabular format, could also be an efficient way for us to align on the logic. A few examples are provided above.

You can submit your suggested as a commit suggestion if you'd like - this gives us a way to click a single button to commit your proposed change into the PR.

Reply:
```suggestion

```

Reply:
[marking as resolved - let's move outstanding todos / clarifications to the [v2 PR](https://github.com/ArbitrumFoundation/docs/pull/230)]